src/test/java/cmput402/tdd/AppTest.java

```diff
@@ -265,4 +265,269 @@ public void testDisplayPeople() {
         assertEquals(app.displayPeople(), out);
         
     }
+  
+    @Test
+    public void testEditRecipe(){
+        App app = new App();
+        Recipe recipe = new Recipe("apple pie");
+
+        Item item1 = new Item("apple", 4.0f);
+        Item item2 = new Item("sugar", 2.0f);
+
+        //Test 1: Add new item to recipe
+        ByteArrayInputStream in = new ByteArrayInputStream("add 1\napple\n4.0".getBytes());
+        Scanner scanner = new Scanner(in);
+        try {
+            app.editRecipe(scanner, recipe);
+        }
+        catch(Exception e){
+            Assert.fail();
```

Comment:
The import statement to import the fail method should specify the package it comes from. No need to explicitly add it here
```suggestion
            fail();
```

Reply:
Changed in latest commit